packages/scripts/scripts/lint-style.js

```diff
@@ -11,12 +11,15 @@ const {
 	fromConfigRoot,
 	getCliArgs,
 	hasCliArg,
+	hasFileInCliArgs,
 	hasProjectFile,
 	hasPackageProp,
 } = require( '../utils' );
 
 const args = getCliArgs();
 
+const defaultFilesArgs = ! hasFileInCliArgs ? [ '**/*.{css,scss}' ] : [];
```

Comment:
`**/*.{css,scss}` - I'm not convinced about this proposal so I'm happy to revisit based on the feedback.

I also consider myself: `src/**/*.{css,scss}` to be ready for integration with `wp-scripts build`. We can always revisit later once we have #14801 implemented.

Reply:
In #15977, I propose we ignore `build` and `node_modules` folders by default, so this should be less important than at the moment.